panic on pixel 6 pro(android13)

**Describe the bug**
在开了oem,并且root后的真机pixel6pro上执行./ecapture tls -iwlan0 -k ./ecapture_openssl_key.log -m key后，打开华为应用商店panic

`2024-05-25T14:05:40Z ERR master secret length is too long, truncate to 64 bytes, but it may cause keylog file error length=79
panic: runtime error: index out of range [64] with length 64

goroutine 39 [running]:
github.com/gojue/ecapture/user/module.(*MOpenSSLProbe).mk13NullSecrets(_, _, {0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...}, ...)
	/home/runner/work/ecapture/ecapture/user/module/probe_openssl.go:594 +0x124
github.com/gojue/ecapture/user/module.(*MOpenSSLProbe).bSSLEvent13NullSecrets(...)
	/home/runner/work/ecapture/ecapture/user/module/probe_openssl.go:563
github.com/gojue/ecapture/user/module.(*MOpenSSLProbe).saveMasterSecretBSSL(0x4000459500, 0x4000a33400)
	/home/runner/work/ecapture/ecapture/user/module/probe_openssl.go:504 +0x380
github.com/gojue/ecapture/user/module.(*MOpenSSLProbe).Dispatcher(0x0?, {0xe4e8a0?, 0x4000a33400?})
	/home/runner/work/ecapture/ecapture/user/module/probe_openssl.go:630 +0x84
github.com/gojue/ecapture/user/module.(*Module).Dispatcher(0x4000459500, {0xe4e8a0, 0x4000a33400})
	/home/runner/work/ecapture/ecapture/user/module/imodule.go:387 +0x144
github.com/gojue/ecapture/user/module.(*Module).perfEventReader.func1()
	/home/runner/work/ecapture/ecapture/user/module/imodule.go:299 +0x178
created by github.com/gojue/ecapture/user/module.(*Module).perfEventReader in goroutine 1
	/home/runner/work/ecapture/ecapture/user/module/imodule.go:267 +0x1e0`

**To Reproduce**
Steps to reproduce the behavior:
1. adb shell
2. su
3. cd /data/local/tmp
4. ./ecapture tls -iwlan0 -k ./ecapture_openssl_key.log -m key
5. 打开华为应用商店

**Expected behavior**
it works

**Screenshots**
![image](https://github.com/gojue/ecapture/assets/62824352/e878e247-2a5c-4880-82cf-3adae704674d)


**Linux Server/Android (please complete the following information):**

- Env:  android真机
- OS:  pixel6 pro  TQ3A.230805.001
- Arch:  aarch64
- Kernel Version:  5.10(android 13)
- Version: Linux localhost 5.10.157-android13-4-00003-g776d0a76f6aa-ab10208116 #1 SMP PREEMPT Thu May 25 12:11:12 UTC 2023 aarch64 Toybox

**Additional context**
浏览https网页没有问题

--- a/user/module/imodule.go
+++ b/user/module/imodule.go
@@ -142,7 +142,7 @@ func (m *Module) autoDetectBTF() {
 			m.isCoreUsed = true
 		}
 	} else {
-		m.logger.Warn().Err(err).Msg("Failed to detect container environment, error:%v,This may cause eCapture not to work.\n" + BtfNotSupport)
+		m.logger.Warn().Err(err).Msg("Failed to detect container environment,This may cause eCapture not to work.\n" + BtfNotSupport)
 	}
 }
 func (m *Module) geteBPFName(filename string) string {

--- a/user/module/probe_openssl.go
+++ b/user/module/probe_openssl.go
@@ -589,6 +589,11 @@ func (m *MOpenSSLProbe) mk13NullSecrets(hashLen int,
 ) bool {
 	isNUllCount := 5
 
+	// The mandatory setting is 64 bytes, and the length of the XXXX secret is not allowed to be more than 64 bytes.
+	if hashLen > 64 {
+		hashLen = 64
+	}
+
 	var chsChecked, ctsChecked, shsChecked, stsChecked, esChecked bool
 	for i := 0; i < hashLen; i++ {
 		if !chsChecked && ClientHandshakeSecret[i] != 0 {

